### PR TITLE
Remove getOrCreate from MeiliSearchClient

### PR DESCRIPTION
**Pull request by brunoocasali**
As the title says, the idea is to remove `getOrCreate` method accordingly to this https://github.com/meilisearch/integration-guides/issues/157#issuecomment-998826170

Related to MeiliSearch v0.25 https://github.com/meilisearch/integration-guides/issues/157

### CODE REVIEW

**Comment by curquiza** [comment]
Thank you so much Bruno for this help!

I changed the based branch, should be `bump-meilisearch-v0.25.0` and not `main` 🙂

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -235,6 +235,10 @@ settings_guide_searchable_1: |-
 settings_guide_displayed_1: |-
   await client.index('movies').updateSettings(IndexSettings(
       displayedAttributes: ['title', 'description', 'genre', 'release_date']));
+settings_guide_sortable_1: |-
+  await client
+      .index('books')
+      .updateSettings(IndexSettings(sortableAttributes: ['author', 'price']));
```
Comment (curquiza) [inline .code-samples.meilisearch.yaml:241]: Should be in another PR :)

**Review by curquiza — CHANGES_REQUESTED** [review]

**Comment by brunoocasali** [comment]
> Thank you so much Bruno for this help!
> 
> I changed the based branch, should be `bump-meilisearch-v0.25.0` and not `main` 🙂 Could you rebase the branch with `bump-meilisearch-v0.25.0` becasue now there is the Amélie's commit in your PR

Oh ok, awesome! I'll do that!

So just to clarify we must create multiple PR's pointing to the `bump-meilisearch-v0.25.0`, once meili releases the new version and the integration sdk is good to go, we merge + release the PR. 
But the idea is to always create a PR to `bump-meilisearch-v0.25.0` instead of committing directly to it right? (to be able to review small pieces before the merge).

**Comment by curquiza** [comment]
>  So just to clarify we must create multiple PR's pointing to the bump-meilisearch-v0.25.0, once meili releases the new version and the integration sdk is good to go, we merge + release the PR.

Yes, we do the changes regarding vX.X.X of MeiliSearch on the `bump-meilisearch-vX.X.X` branch. This way, we can keep releasing new versions on `main` of we find out any bugs on `main`

**Comment by curquiza** [comment]
> So just to clarify we must create multiple PR's pointing to the bump-meilisearch-v0.25.0, once meili releases the new version and the integration sdk is good to go, we merge + release the PR.

Yes it's cleaner since the PR title will be put in the changelog of the release, it will help us create the release changelog at the end. Plus, it's easier to review :)

**Review by curquiza — APPROVED** [review]
LGTM! :)
